### ISSUE TITLE
When Local and Remote Desktop are different sizes and in Quake mode, permanently takes on smaller screen size

### ISSUE THREAD

**Issue by ConEmu** [ConEmu/old-issues#1100]
_From [msumer...@gmail.com](https://code.google.com/u/118120358928560564340/) on June 18, 2013 07:08:34_

OS version: Win7 SP1 x64 ConEmu version: 130427 x64 _Bug description_ When I use ConEmu locally, then go to another machine and remote desktop (RDP) into my machine, ConEmu resizes itself to the smaller screen.  When I log back in locally, it does not reset itself to the larger screen.  I use Quake mode.  So, I'm not sure if this is only an issue with Quake mode or an issue in general.  The only way I found to fix it is to restart ConEmu . _Steps to reproduction_ 1. Run ConEmu in Quake mode while logged in locally.
2. Go to another machine with a smaller screen, remote desktop to machine, and observe ConEmu .
3. Log back into machine locally and observe that ConEmu screen has been resized to smaller screen and doesn't fix itself.

_Original issue: http://code.google.com/p/conemu-maximus5/issues/detail?id=1101_

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 18, 2013 07:31:59_

Try latest alpha build. There was resize bug fixes, maybe your was fixed too.

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on June 20, 2013 09:23:59_

I believe you've not setup size of window relatively to screen width. For example, 80%
In that case (some fixed width in cells) any changes of screen resolution are treated as "user invoked" and are stored in the current session parameters.

**Comment by ConEmu** [comment]
_From [msumer...@gmail.com](https://code.google.com/u/118120358928560564340/) on June 26, 2013 20:23:11_

I don't think I explained myself the first time around.  It's not that the ConEmu window doesn't resize.  It's the console inside of ConEmu that does not resize back to my local screen size.  So, for example, I have a 24" monitor with 1900x1200 resolution locally (say that's ~120 characters wide in a console).  I RDP from a machine (full screen) to my own on a 22" monitor with only a 1650x1050 resolution (say ~80 characters wide).  When RDP'ed, it resizes and I have 80 characters wide in the window.  Log back in locally, ConEmu has resized to the 1900x1200 resolution, but the text in the console stays at the 80 character size and is either left aligned or centered (I've seen both).  Note that the majority of the time, I am using a PowerShell console.  I rarely use a standard console with ConEmu .

Now, since I've started using the newer Previews (thank you for starting to release these) and bleeding edge at home, I have not seen these issues arise.  I will attempt to reproduce this, but I can only use the Preview version at work (see my other ticket on false-positives with Symantec).  At this point, I'd recommend closing this ticket until I can reproduce it on the latest builds.
